docs/Administration.rst

```diff
@@ -2456,18 +2460,49 @@ should be sent to you.
 Administrating spaces 
 ======================
 
-It is possible to restrict which users can create and/or manage spaces
+It is possible to restrict which users can create and/or manage spaces and templates
 through an easy to use interface .
 
 |image305|
 
-.. note:: By default, only the super user (i.e. **root**) is allowed to create spaces and only administrators (i.e.
-          the members of the group ``*:/platform/administrators``) are allowed to manage spaces.
+.. note:: By default, only the super user (i.e. **root**) is allowed to create spaces and templates and only administrators (i.e.
+          the members of the group ``*:/platform/administrators``) are allowed to manage spaces and templates.
           
 The spaces administration interface is accessible via the menu Administration --> Spaces.
 
 |image306|
 
+This Spaces menu contains two choices:
+
+- "Manage Templates" : Lets an administrator display all space templates available 
+- "Manage Spaces" : The administrator tab displays the list of groups who can :ref:`Create <Create-Space>` or :ref:`Manage <AdministratingSpaces.ManageSpaces>` spaces.
+
+
+
+.. note:: the list of templates is read-only. Changing a template's details requires to edit the config file as explained here.
```

Comment:
```suggestion
.. note:: the list of templates is read-only. Changing a template details requires to edit the config file as explained in developers guide.
```
@hghorbel please adding link to relevant section on developers guide

Reply:
Done in a new commit.